onnx/backend/test/case/node/and.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from .base import Base, expect
+
+
+class And(Base):
+
+    @staticmethod
+    def export():
+        node = onnx.helper.make_node(
+            'And',
+            inputs=['x', 'y'],
+            outputs=['and'],
+        )
+
+        x = (np.random.randn(3, 4, 5) > 0).astype(np.bool)
+        y = (np.random.randn(3, 4, 5) > 0).astype(np.bool)
+        z = np.logical_and(x, y)
+        expect(node, inputs=[x, y], outputs=[z],
```

Comment:
Thank you very much! Would you also help to add cases to cover,
1. broadcast not specified, and two inputs have different shape.
2. cases for different attribute values of "axis".